GenX/README.md

```diff
@@ -16,3 +16,6 @@ GenX is licensed under the [GPL v2 license](https://github.com/GenXProject/GenX.
 
 The example cases in this directory are taken from
 [https://github.com/GenXProject/GenX.jl/tree/v0.4.1/example_systems](https://github.com/GenXProject/GenX.jl/tree/v0.4.1/example_systems).
+
+Cases were modified by setting `UCommit: 1` to consider unit commitment of operation
+variables, which is the source of integrality in GenX.
```

Comment:
You did't commit these?

Reply:
not yet, it is failing to write the files, working on it